scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SchedulerStarter.java

```diff
@@ -142,6 +153,21 @@ private static void start(CommandLine commandLine) throws Exception {
             startRouter();
         }
 
+        String schedulerHome = PASchedulerProperties.SCHEDULER_HOME.getValueAsString();
+        Path hibernateRmConfiguration = Paths.get(schedulerHome, "config", "rm", "database.properties");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "config" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1192)